CustomEntitySearchTest/Constants.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LookupTests
```

Comment:
>LookupTests [](start = 10, length = 11)

AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch.Tests

Reply:
update default namespace, and namespaces of other files as well

---
In reply to: [304660280](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r304660280) [](ancestors = 304660280)